src/OSPSuite.Infrastructure.Import/Services/SensitivityAnalysisRunResultsImportTask.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using OSPSuite.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.SensitivityAnalyses;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Services;
+using OSPSuite.Utility.Events;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Infrastructure.Import.Services
+{
+   public interface ISensitivityAnalysisRunResultsImportTask
+   {
+      Task<SensitivityAnalysisRunResultsImport> ImportResults(IModelCoreSimulation simulation, IReadOnlyCollection<string> files, CancellationToken cancellationToken, bool showImportProgress = true);
```

Comment:
Can import SARunResults from multiple files. This was missing for the SA workflow